detox/src/devices/DeviceRegistry.js

```diff
@@ -2,16 +2,15 @@ const _ = require('lodash');
 const fs = require('fs-extra');
 const plockfile = require('proper-lockfile');
 const retry = require('../utils/retry');
-const environment = require('../utils/environment');
-const DEVICE_LOCK_FILE_PATH = environment.getDeviceLockFilePath();
 
 const LOCK_RETRY_OPTIONS = {retries: 10000, interval: 5};
 
 class DeviceRegistry {
 
-  constructor({getDeviceIdsByType, createDevice}) {
+  constructor({getDeviceIdsByType, createDevice, lockfile}) {
```

Comment:
Everything here is fine per se, but see my comment at the beginning about what is *that good reason* to split lock files into 2, by platform?